src/components/Table/table.scss

```diff
@@ -101,8 +101,24 @@
       }
 
       &-notification {
-        .notification-fail svg > * {
-          fill: $silver;
+        position: relative;
+
+        .notification-fail {
+          position: absolute;
+          top: 9px;
+          left: 14px;
+
+          path {
+            fill: $burntSienna;
+          }
+
+          svg {
+            width: 14px;
+            height: 14px;
+            padding: 1px;
+            background-color: white;
+            border-radius: 50%;
+          }
```

Comment:
`-notification` isn't part of the common style for the table. These styles should be in the `alertsTableRow.scss`

Reply:
Done